static/js/libs/arrays.js

```diff
@@ -0,0 +1,23 @@
+function arraysEqual(oldArray, newArray) {
+  if (oldArray === newArray) {
+    return true;
+  }
+  if (
+    oldArray === null || newArray === null ||
+    (oldArray.length !== newArray.length)) {
+    return false;
+  }
+
+  oldArray.sort();
+  newArray.sort();
+
+  for(let i = 0; i < oldArray.length; i++) {
```

Comment:
Is `sort` needed? If arrays are meant to be equal can't we just check elements one by one without sorting? Or does `arraysEqual` checks not for equality but just if arrays contain exactly the same elements (in any order?) - in which case the name of the function is confusing.

Reply:
The sorting is a catch-all in case there's some sorting elsewhere